core/rend/dx11/dx11_overlay.cpp

```diff
@@ -140,65 +187,72 @@ void DX11Overlay::draw(u32 width, u32 height, bool vmu, bool crosshair)
 			vp.MaxDepth = 1.f;
 			deviceContext->RSSetViewports(1, &vp);
 			quad.draw(vmuTextureViews[i], samplers->getSampler(false));
+			if (config::OnlyShowVMUA1)  // Only break if single VMU mode is enabled
+				break;
 		}
 	}
 	if (crosshair)
 	{
-		for (u32 i = 0; i < config::CrosshairColor.size(); i++)
+		for (size_t i = 0; i < vmuTextures.size(); i++)
```

Comment:
Lightgun crosshairs and VMU displays are different things. This code shouldn't change.